packages/vsce/src/resources/SessionHandler.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { IProfile, IProfileLoaded } from "@zowe/imperative";
+import { imperative } from "@zowe/zowe-explorer-api";
+import { CICSSession } from "./CICSSession";
+import { ISessionHandler } from "./ISessionHandler";
+
+export class SessionHandler implements ISessionHandler {
+  public sessions: Map<String, CICSSession>;
+  session: CICSSession | undefined;
```

Comment:
I'm not sure we need a global `sessions` and `session` object? Think we can remove `session` and just store them all in the map. 

Also, we could make `sessions` private as we have a getter method 😄